<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpadesInputData</class>
 <widget class="QDialog" name="SpadesInputData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure SPAdes Input Type</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinAndMaxSize</enum>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Required input (at least one)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Illumina/Ion Torrent reads</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetMinAndMaxSize</enum>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>-1</number>
           </property>
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Sequencing platform</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="sequencingPlatformComboBox">
             <item>
              <property name="text">
               <string>Illumina</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ion Torrent</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinAndMaxSize</enum>
           </property>
           <property name="bottomMargin">
            <number>4</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMinAndMaxSize</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QCheckBox" name="pairEndCheckBox">
                 <property name="text">
                  <string>Paired-end reads</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QComboBox" name="pairEndReadsDirectionComboBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <item>
                    <property name="text">
                     <string>fr</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>rf</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>ff</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="pairEndReadsTypeComboBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <item>
                    <property name="text">
                     <string>Separate reads</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Interlaced reads</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QCheckBox" name="hightQualityCheckBox">
                 <property name="text">
                  <string>High-quality mate-pairs</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                 <item>
                  <widget class="QComboBox" name="hightQualityReadsDirectionComboBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <item>
                    <property name="text">
                     <string>fr</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>rf</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>ff</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="hightQualityReadsTypeComboBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <item>
                    <property name="text">
                     <string>Separate reads</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Interlaced reads</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <widget class="QCheckBox" name="unpairedReadsCheckBox">
               <property name="text">
                <string>Unpaired reads</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="leftMargin">
         <number>7</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QCheckBox" name="pacBioCcsCheckBox">
          <property name="text">
           <string>PacBio CCS reads</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>98</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Additional input</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>18</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Illumina/Ion Torrent reads</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>100</number>
           </property>
           <item>
            <widget class="QCheckBox" name="matePairsCheckBox">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Mate-pairs</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QComboBox" name="matePairsReadsDirectionComboBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>fr</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>rf</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ff</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="matePairsTypeComboBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <item>
                <property name="text">
                 <string>Separate reads</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Interlaced reads</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>18</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="leftMargin">
         <number>8</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QCheckBox" name="pacBioClrCheckBox">
          <property name="text">
           <string>PacBio CLR reads</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="oxfordNanoporeCheckBox">
          <property name="text">
           <string>Oxford Nanopore reads</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="sangerReadsCheckBox">
          <property name="text">
           <string>Sanger reads</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="trustedContigsCheckBox">
          <property name="text">
           <string>Trusted contigs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="untrustedContigsCheckBox">
          <property name="text">
           <string>Untrusted contigs</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpadesInputData</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpadesInputData</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>matePairsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>matePairsReadsDirectionComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>matePairsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>matePairsTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pairEndCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pairEndReadsDirectionComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pairEndCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pairEndReadsTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hightQualityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hightQualityReadsDirectionComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hightQualityCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hightQualityReadsTypeComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
